tests/window_management_tests/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+include(CMakeDependentOption)
+
+include_directories(
+  ${CMAKE_SOURCE_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}
+  ${PROJECT_SOURCE_DIR}/src/include/platform
+  ${PROJECT_SOURCE_DIR}/src/include/cookie
+  ${PROJECT_SOURCE_DIR}/src/include/common
+  ${PROJECT_SOURCE_DIR}/src/include/server
+  ${PROJECT_SOURCE_DIR}/src/include/client
+  ${PROJECT_SOURCE_DIR}/src/include/gl
+  ${PROJECT_SOURCE_DIR}/include/renderers/sw
+)
+
+set(WINDOW_MANAGEMENT_TESTS_SRCS
+  test_minimal_window_manager.cpp
+)
+
+link_directories(${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+mir_add_wrapped_executable(mir_window_management_tests NOINSTALL
+  ${WINDOW_MANAGEMENT_TESTS_SRCS}
+  ${MIR_SERVER_OBJECTS}
+  ${MIR_PLATFORM_OBJECTS}
```

Comment:
Are these useful? The corresponding shared libraries are already linked implicitly which risks having two definitions at runtime